tlatools/org.lamport.tlatools/test-model/Github1145.tla

```diff
@@ -0,0 +1,14 @@
+\* Copyright (c) 2025, Oracle and/or its affiliates.
+---- MODULE Github1145 ----
+
+EXTENDS Naturals, Sequences
+
+F(seq) == SubSeq(seq, 2, 2)
```

Comment:
(for my own understanding) why is this operator necessary? Is it that the functions will not otherwise be coerced to a sequence?

Reply:
It just helps reduce duplication. I believe the test will still fail in current master if you inline `F`.